common/OpTestQemu.py

```diff
@@ -345,15 +378,25 @@ def __init__(self, conf=None, qemu_binary=None, pnor=None, skiboot=None,
         self.system = None
 
     def __del__(self):
+<<<<<<< HEAD
```

Comment:
:-1:

Reply:
confused.  I don't have this in my tree, github says "outdated".

Reply:
oh, it's in the pep8 patch and it gets fixed in the next patch.  whoops!  i'll fix and rebase